commands/management/remove.go

```diff
@@ -68,6 +72,13 @@ func (cmd *removeMySQLCommand) serviceType() *string {
 	return nil
 }
 
+func (cmd *removeMySQLCommand) validateParams() error {
+	if cmd.ServiceID == "" && cmd.ServiceName == "" {
+		return fmt.Errorf("You need to specify --service-id or --service-name")
```

Comment:
pmm-managed also does not allow both: https://github.com/percona/pmm-managed/blob/91f6507da2959d972987b09a32cd710afdfaffe4/services/management/service.go#L116-L124 That's why, in general, we prefer server-side validation to client-side.

On the other side, pmm-managed does not know pmm-admin flag names, so an error message can't be perfect.

I don't know the general solution, but in that case, let's sync client-side validation implementation with pmm-managed's, and add a comment reminding us to do that on the next change.

Reply:
Since pmm-managed is already doing the validation I just wanted to add a pre-validation to avoid an unnecessary request but you are right, it is better to have validations just in one place.. I am going to remove this one in favor of pmm-managed's validation.